cli-plugins/manager/manager.go

```diff
@@ -40,17 +40,27 @@ func (e errPluginNotFound) Error() string {
 //
 // [ConfigFile.CLIPluginsExtraDirs]: https://pkg.go.dev/github.com/docker/cli@v26.1.4+incompatible/cli/config/configfile#ConfigFile.CLIPluginsExtraDirs
 func getPluginDirs(cfg *configfile.ConfigFile) []string {
+	var configPluginDirs []string
 	var pluginDirs []string
-
 	if cfg != nil {
-		pluginDirs = append(pluginDirs, cfg.CLIPluginsExtraDirs...)
+		configPluginDirs = append(configPluginDirs, cfg.CLIPluginsExtraDirs...)
 	}
+	pluginDirs = append(pluginDirs, expandEnvironmentVariablesInPluginDirString(configPluginDirs)...)
```

Comment:
No need for the extra variable I think?
And we can also make the function name shorter
```suggestion
		pluginDirs = append(pluginDirs, expandEnvVars(cfg.CLIPluginsExtraDirs)...)
}
```